llvm/test/CodeGen/AMDGPU/do-not-fold-copy.mir

```diff
@@ -0,0 +1,36 @@
+# RUN: llc -mtriple=amdgcn -mcpu=gfx900 -run-pass=si-fold-operands -o - %s | FileCheck %s
+
+---
+liveins:
+name:            do_not_fold_copy_with_implicit_exec
+tracksRegLiveness: true
+body:             |
+  ; CHECK: bb.0:
+  ; CHECK: bb.1:
+  ; CHECK: %[[A:[0-9]*]]:sreg_32 = S_ADD_I32
+  ; CHECK: COPY %[[A]]
+  ; CHECKL SI_LOOP
```

Comment:
Presumably this should be `CHECK:`?
Any reason not to auto-generate test checks?

Reply:
Thanks, this is a fragment of real code from more than one game.
>Any reason not to auto-generate test checks?

I can auto-generate. I just wanted to have a minimal test.

Reply:
I think it is preferable to autogenerate this, so we can see if/when ISA changes in future.
(We may want to re-enable this fold in future with some kind of EXEC analysis.)

Reply:
I auto-generated this test in this commit: https://github.com/llvm/llvm-project/pull/136003/commits/c9a9f10090f1d20b31ac8d32069e2097c4bc1bf6